src/script/engine.zig

```diff
@@ -309,6 +311,27 @@ pub const Engine = struct {
         try self.stack.pushByteArray(first);
     }
 
+    // OP_2OVER: Copies the pair of items two spaces back in the stack to the front
+    fn op2Over(self: *Engine) !void {
+        const stack_length = self.stack.len();
+        const last_element1 = try self.stack.peek(stack_length - 1);
+        const last_element2 = try self.stack.peek(stack_length - 2);
+        try self.stack.pushByteArray(last_element1);
+        try self.stack.pushByteArray(last_element2);
```

Comment:
This implem only work because you have 4 elems in your stack on your test.

`stack_length - 1` give you the very last element of the stack.
What you are looking for is the fourth element.

```
const fourth = try self.stack.peek(3);
const third = try self.stack.peek(2);
try self.stack.pushByteArray(fourth);
try self.stack.pushByteArray(third);
````